### DISCUSSION TITLE
capacitor not working old version iphone

### DISCUSSION THREAD

**Discussion by ManiSabarish9290** [ionic-team/capacitor#4595] [Q&A]
hi i have implemented capacitor in my application. this frame work working latest iphone version. but old iphone version not working. how to solve this issue

**Comment by thomasvidas** [comment]
Capacitor 2 supports iOS 11 through 14 while Capacitor 3 supports iOS 12+

If you look at [this chart for worldwide iOS distribution](https://david-smith.org/iosversionstats/) There is only a difference of 0.4% of people that are on iOS 11 or higher vs iOS 12 or higher. If you _need_ iOS 11 support, you will need to be on Capacitor 2.

**Reply by ManiSabarish9290** [reply]
okay. which capacitor support for all iphone version?

**Reply by ManiSabarish9290** [reply]
iphone version 14.5 working fine but 14.2 not working. below version not working. latest version only working fine.

**Reply by ManiSabarish9290** [reply]
pod 'Capacitor', :path => '../../node_modules/@capacitor/ios'
  pod 'CapacitorCordova', :path => '../../node_modules/@capacitor/ios'


this two pods only am implemented in my application

**Reply by ManiSabarish9290** [reply]
how to implement capacitor 2 in my ios application

**Reply by thomasvidas** [reply]
If you are having a specific problem  please open an issue with a github repo that allows us to reproduce the issue, otherwise there's no way for us to figure out this specific problem you're running into
